scripts/falco-driver-loader

```diff
@@ -711,6 +707,13 @@ if [ -z "$source_only" ]; then
 		exit 1
 	fi
 
+	get_target_id
+	res=$?
+	if [ $res != 0 ]; then
+		>&2 echo "Detected an unsupported target system, please get in touch with the Falco community. Trying to compile anyway."
+		ENABLE_DOWNLOAD=
```

Comment:
```suggestion
		ENABLE_DOWNLOAD="no"
```

Reply:
Ehy Leo! It would be great if this worked :D Unfortunately, in the script we check if it is not emtpy:
```
if [ -n "$ENABLE_DOWNLOAD" ]; then
```
I wanted to make the patch with the smallest diff possible, therefore i avoided to touch it.